src/components/CompareResults/CompareResultsTable.tsx

```diff
@@ -12,15 +12,14 @@ import type { CompareResultsTableHeaders } from '../../types/types';
 import CompareResultsTableRow from './CompareResultsTableRow';
 
 const tableHeaders: CompareResultsTableHeaders[] = [
-  'Platform',
-  'Graph',
-  'Suite',
-  'Test Name',
-  'Base',
-  'New',
-  'Delta',
-  'Confidence',
-  'Total Runs',
+  { id: 'platform', label: 'Platform', align: 'center' },
+  { id: 'graph', label: 'Graph', align: 'center' },
+  { id: 'test-name', label: 'Test Name', align: 'left' },
+  { id: 'base-value', label: 'Base', align: 'center' },
+  { id: 'new-value', label: 'New', align: 'center' },
+  { id: 'delta-percent', label: 'Delta', align: 'center' },
+  { id: 'confidence', label: 'Confidence', align: 'center' },
+  { id: 'total-runs', label: 'Total Runs', align: 'center' },
```

Comment:
hey, 

[question] : is there a reason why only test name is left?
How about making `align={header.align || 'center'}`

Reply:
Original alignment is left for suite and test name and I've left it as it was before.